php-frontend/src/test/java/org/sonar/plugins/php/api/cfg/ControlFlowGraphTest.java

```diff
@@ -39,11 +39,7 @@
 import org.sonar.plugins.php.api.tree.declaration.MethodDeclarationTree;
 import org.sonar.plugins.php.api.tree.expression.AssignmentExpressionTree;
 import org.sonar.plugins.php.api.tree.expression.FunctionExpressionTree;
-import org.sonar.plugins.php.api.tree.statement.BlockTree;
-import org.sonar.plugins.php.api.tree.statement.EchoTagStatementTree;
-import org.sonar.plugins.php.api.tree.statement.ExpressionStatementTree;
-import org.sonar.plugins.php.api.tree.statement.InlineHTMLTree;
-import org.sonar.plugins.php.api.tree.statement.StatementTree;
+import org.sonar.plugins.php.api.tree.statement.*;
```

Comment:
Please configure your IDE to avoid using `import *`: https://github.com/SonarSource/sonar-developer-toolset